tools/generator/src/PBXProductType+Extensions.swift

```diff
@@ -36,6 +36,41 @@ extension PBXProductType {
         }
     }
 
+    var isExecutable: Bool {
+        switch self {
+        case .application: return true
+        case .framework: return true
+        case .staticFramework: return false
+        case .xcFramework: return true // Could be?
```

Comment:
lol

Reply:
I should remove the `?`. XCFrameworks can be static or dynamic.